tests/media/test.txt

```diff
@@ -0,0 +1 @@
SNIFF = True\nSECURE_HSTS_INCLUDE_SUBDOMAINS = True\nSECURE_HSTS_PRELOAD = True\nSECURE_HSTS_SECONDS = 60\nSECURE_PROXY_SSL_HEADER = "(\'HTTP_X_FORWARDED_PROTO\', \'https\')"\nSECURE_REDIRECT_EXEMPT = []\nSECURE_REFERRER_POLICY = \'same-origin\'\nSECURE_SSL_HOST = None\nSECURE_SSL_REDIRECT = True\nSERVER_EMAIL = \'noreply@compredict.ai\'\nSESSION_CACHE_ALIAS = \'default\'\nSESSION_COOKIE_AGE = 1209600\nSESSION_COOKIE_DOMAIN = None\nSESSION_COOKIE_HTTPONLY = True\nSESSION_COOKIE_NAME = \'sessionid\'\nSESSION_COOKIE_PATH = \'/\'\nSESSION_COOKIE_SAMESITE = \'Lax\'\nSESSION_COOKIE_SECURE = True\nSESSION_ENGINE = \'django.contrib.sessions.backends.db\'\nSESSION_EXPIRE_AT_BROWSER_CLOSE = False\nSESSION_FILE_PATH = None\nSESSION_SAVE_EVERY_REQUEST = False\nSESSION_SERIALIZER = \'django.contrib.sessions.serializers.JSONSerializer\'\nSETTINGS_MODULE = \'config.settings.production\'\nSHORT_DATETIME_FORMAT = \'m/d/Y P\'\nSHORT_DATE_FORMAT = \'m/d/Y\'\nSIGNING_BACKEND = \'django.core.signing.TimestampSigner\'\nSILENCED_SYSTEM_CHECKS = []\nSITE_ID = 1\nSTATICFILES_DIRS = [\'/app/computation_server/static\']\nSTATICFILES_FINDERS = [\'django.contrib.staticfiles.finders.FileSystemFinder\', \'django.contrib.staticfiles.finders.AppDirectoriesFinder\', \'djangobower.finders.BowerFinder\']\nSTATICFILES_STORAGE = \'whitenoise.storage.CompressedManifestStaticFilesStorage\'\nSTATIC_ROOT = \'/app/static\'\nSTATIC_URL = \'/static/\'\nTEMPLATES = [{\'BACKEND\': \'django.template.backends.django.DjangoTemplates\', \'DIRS\': [\'/app/computation_server/templates\'], \'OPTIONS\': {\'debug\': True, \'loaders\': [(\'django.template.loaders.cached.Loader\', [\'django.template.loaders.filesystem.Loader\', \'django.template.loaders.app_directories.Loader\'])], \'context_processors\': [\'django.template.context_processors.debug\', \'django.template.context_processors.request\', \'django.contrib.auth.context_processors.auth\', \'django.template.context_processors.i18n\', \'django.template.context_processors.media\', \'django.template.context_processors.static\', \'django.template.context_processors.tz\', \'django.contrib.messages.context_processors.messages\']}}]\nTESTING = False\nTEST_NON_SERIALIZED_APPS = []\nTEST_RUNNER = \'django.test.runner.DiscoverRunner\'\nTHIRD_PARTY_APPS = [\'crispy_forms\', \'allauth\', \'allauth.account\', \'rest_framework\', \'djangobower\', \'rangefilter\']\nTHOUSAND_SEPARATOR = \',\'\nTIME_FORMAT = \'P\'\nTIME_INPUT_FORMATS = [\'%H:%M:%S\', \'%H:%M:%S.%f\', \'%H:%M\']\nTIME_ZONE = \'Europe/Berlin\'\nUSE_I18N = True\nUSE_L10N = True\nUSE_THOUSAND_SEPARATOR = False\nUSE_TZ = True\nUSE_X_FORWARDED_HOST = False\nUSE_X_FORWARDED_PORT = False\nWSGI_APPLICATION = \'config.wsgi.application\'\nX_FRAME_OPTIONS = \'DENY\'\nYEAR_MONTH_FORMAT = \'F Y\'\n\n\nYou\xe2\x80\x99re seeing this error because you have DEBUG = True in your\nDjango settings file. Change that to False, and Django will\ndisplay a standard page generated by the handler for this status code.\n\n'
```

Comment:
The file contains some information about computation server I would rather than save it in a file, could you please remove all the configuration from aicore base and production settings from the file and keep the rest as it is, it still should work.

Reply:
Yes, This is only about checking if json would be able to decode it.